guides/common/modules/proc_attaching-red-hat-subscriptions-to-content-hosts.adoc

```diff
@@ -5,17 +5,14 @@ Using activation keys is the main method to attach subscriptions to content host
 However, an activation key cannot update an existing host.
 If you need to attach new or additional subscriptions, such as future-dated subscriptions, to one host, use the following procedure.
 
-ifndef::orcharhino[]
 include::snip_only-valid-if-sca-disabled.adoc[]
-endif::[]
 
 For more information about updating multiple hosts, see xref:Updating_Red_Hat_Subscriptions_on_Multiple_Hosts_{context}[].
 
 For more information about activation keys, see xref:Managing_Activation_Keys_{context}[].
 
 .{Project} Subscriptions
-
-In {Project}, you must maintain a {RHEL} {Project} subscription, formerly known as {RHEL} Smart Management, for every {RHEL} host that you want to manage.
+In {Project}, you must maintain a {RHEL} Satellite subscription, formerly known as {RHEL} Smart Management, for every {RHEL} host that you want to manage.
```

Comment:
I am unsure about the product name here. Can someone please ACK this? See also https://access.redhat.com/documentation/en-us/red_hat_satellite/6.13/html/managing_content/managing_red_hat_subscriptions_content-management#Attaching_Red_Hat_Subscriptions_to_Content_Hosts_content-management

Reply:
Seems fine to me unless someone else has different opinion.